js/piwik.js

```diff
@@ -5148,9 +5219,9 @@ if (typeof Piwik !== 'object') {
                  * @param string name (optional) The Event's object Name (a particular Movie name, or Song name, or File name...)
                  * @param float value (optional) The Event's value
                  */
-                trackEvent: function (category, action, name, value) {
+                trackEvent: function (category, action, name, value, callbacks) {
```

Comment:
Is `callbacks` a common name for that? Reading that parameter I expected to pass an array of callbacks. Also it needs to be documented as a parameter. Shall we maybe rather still use `callback`, especially since a function callback would work as well? Or is it maybe worth to use a `settings` object to allow `{success: function()..., error: function ()...}` that would be okay for me as it is optional anyway. Just to avoid having to add many more optional parameters in the future.